doc/api/dgram.md

```diff
@@ -107,8 +107,8 @@ properties.
 added: v0.1.99
 -->
 
-* `port` {number} - Integer, Optional
-* `address` {string}, Optional
+* `port` {number} Integer, Optional.
```

Comment:
{integer} Optionality is implied by the signature, so no need to write it again here.

Ditto rest of the file.